sbpy/exceptions.py

```diff
@@ -22,3 +22,7 @@ class SbpyWarning(Warning):
 
 class OptionalPackageUnavailable(SbpyWarning):
     "Optional package is not available."
+
+
+class TestingNeeded(SbpyWarning):
+    "More testing is needed to understand the issue."
```

Comment:
Looks good

Reply:
Thanks, @migueldvb .  Was this comment for this point specifically or the PR as a whole?